docs/jetstream/operations.md

```diff
@@ -58,6 +58,15 @@ Argo provides a Web UI to access running workflows. Users need to authenticate u
 * Open [https://localhost:2746](https://localhost:2746)
 * Use the generated Bearer token (including the word `Bearer`) for authentication
 
+### Deleting Old Workflows
+
+* The Workflow UI might get less responsive the more workflows have been run in the past
+* To delete workflows that are older than 4 days run:
+
+```
+kubectl get wf -o go-template -n argo --template '{{range .items}}{{.metadata.name}} {{.metadata.creationTimestamp}}{{"\n"}}{{end}}' | awk '$2 <= "'$(gdate -d '4 days ago' -Ins --utc | sed 's/+0000/Z/')'" { print $1 }' | gxargs --no-run-if-empty kubectl delete wf -n argo
```

Comment:
Does this command delete all workflows on the cluster older than 4 days? If so, could we target specific subsets like Jetstream vs population sizing? If not, do we have a way to target all?

Reply:
yeah it's deleting everything older than 4 days. There might be a way to apply a regex to filter out jetstream workflows. Or we could just increase the time span to everything older than a month

Reply:
```suggestion
kubectl get wf -o go-template -n argo --template '{{range .items}}{{.metadata.name}} {{.metadata.creationTimestamp}}{{"\n"}}{{end}}' | awk '$2 <= "'$(gdate -d '30 days ago' -Ins --utc | sed 's/+0000/Z/')'" { print $1 }' | gxargs --no-run-if-empty kubectl delete wf -n argo
```

I'll just increase the threshold to 30 days which should still be totally fine. It was just getting slow with having 3 years of workflow data.